services/.env.example

```diff
@@ -101,6 +101,8 @@ TWILIO_PROGRAM_TOKEN_SECRET=
 TWILIO_AUTHTOKEN=
 TWILIO_WHATSAPP_NUMBER=
 TWILIO_MESSAGING_SID=MG
+TWILIO_TEST_FROM_NUMBER_VOICE=+31600000000
+MOCK_TWILIO=True/False
```

Comment:
If you use 'real' Boolean values here(like `TRUE` or `1`) `dotenv` will convert it to real Boolean variables; so you don't need to compare the string like below.

Reply:
@elwinschmitz I tried this, but this doesn't work for me.. When I have MOCK_TWILIO=False and use `if (process.env.MOCK_TWILIO)` then it still evaluates to true.. (in line with it being read as a string and not as a boolean).

Which is also what I remembered from earlier occurrences, which is why I used the string comparison now

Reply:
Hmm.. I thought that I committed it, but now I can't seem to find it anymore..

I think because the values `0`, `1`, `TRUE`, `FALSE` are also valid Javascript, but `False` isn't valid JS and therefore, 'just a string'.

So I added a comment to the `.env.example` file etc about it... But it got lost in some conflict-resolving/rebaseing or something... 😕 

Nevermind for now. ;)